### PR TITLE
Sweets/opensea

### PR DESCRIPTION
**Pull request by sweetmantech**
# OpenSea Name & Description
- Name ✅ 
- Description: `{name} generated by @{twitterHandle}` ✅ 

### Example:
https://testnets.opensea.io/assets/mumbai/0xfd3e5624332f1865cf02b89da70c4d1fdc41bc8d/189
Screenshot 2022-12-23 at 10 11 16 AM
Screenshot 2022-12-23 at 10 11 26 AM

### CODE REVIEW

File: components/MintButton/MintButton.tsx [hunk 1]
```diff
@@ -58,15 +60,18 @@ const MintButton: FC<MintButtonProps> = ({ twitterHandle }) => {
     const file = dataURLtoFile(dataUri, "a.png")
     const ipfsUrl = await storeBlob(file)
 
-    const receipt = (await axios.get(`/api/mint?imageUri=${ipfsUrl}`)) as any
+    const generatedBy = transformTwitterHandle(twitterHandle)
+    const receipt = (await axios.get(
+      `/api/mint?imageUri=${ipfsUrl}&name=${generated}&description=${generated} generated by ${generatedBy}`,
+    )) as any
```
Comment (crazzywizard) [inline components/MintButton/MintButton.tsx]: you can prob refactor this to be more readable
```
const receipt = await axios.get('/api/mint',{params{imageUri:ipfsUrl, name: generated, description:`${generated} generated by ${generatedBy}`}})```

File: lib/transformTwitterHandle.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import * as _ from "lodash"
+
+const transformTwitterHandle = (twitterHandle?: string): string => {
+    if (twitterHandle) {
+      return _.startsWith(twitterHandle, "@", 0) ? twitterHandle : `@${twitterHandle}`
+    }
+    return "anonymous"
+}
+
+export default transformTwitterHandle
```
Comment (crazzywizard) [inline lib/transformTwitterHandle.ts]: can you add a new line , linter will complain

**Review by crazzywizard — APPROVED** [review]

File: components/MintButton/MintButton.tsx (hunk 1 above)
Comment (sweetmantech) [reply components/MintButton/MintButton.tsx]: updated ✅ 
Screenshot 2022-12-23 at 10 22 38 AM

File: lib/transformTwitterHandle.ts (hunk 2 above)
Comment (sweetmantech) [reply lib/transformTwitterHandle.ts]: should be fixed now? 👀

**Review by crazzywizard — APPROVED** [review]
LGTM!

**Pull request merged**